src/algorithms/DouglasRachford.jl

```diff
@@ -22,7 +22,7 @@ end
 ################################################################################
 # Constructor(s)
 
-function DRSIterator(x0::BlockArray{R}; f=Zero(), g=Zero(), gamma::R=1.0, maxit::I=10000, tol::R=1e-4, verbose=1) where {I, R}
+function DRSIterator(x0::BlockArray{R}; f=Zero(), g=Zero(), gamma::R=1.0, maxit::I=10000, tol::R=1e-4, verbose=2) where {I, R}
```

Comment:
Why `verbose=2` by default? I would say `1`, that is the minimum (nonzero) verbosity.